pkg/lib/cockpit-components-context-menu.tsx

```diff
@@ -47,12 +47,8 @@ export const ContextMenu = ({ parentId, children } : {
         };
 
         const _handleClick = (event: MouseEvent) => {
-            if (event && event.button === 0 && event.target instanceof HTMLElement) {
-                const wasOutside = root.current && !root.current.contains(event.target);
-
-                if (wasOutside)
-                    setVisible(false);
-            }
+            if (event.button === 0)
```

Comment:
But this was originally checked in as:
```
+        const _handleClick = (event) => {
+            if (event && event.button === 0) {
+                const wasOutside = !(event.target.contains === root.current);
+
+                if (wasOutside)
+                    setVisible(false);
+            }
+        };
```

Reply:
as per https://github.com/cockpit-project/cockpit/commit/f9b01663060c5055d22d9639fcd6504d850e5378 `event.target.contains` is a function, and `root.current` is an element, so they're never equal.  The comparison is equivalent to `const wasOutside = true;`